doc/flame/inputs/keyboard-input.md

```diff
@@ -91,6 +91,32 @@ that triggered the callback in the first place. The second is a set of the curre
 The returned value should be `true` to allow the continuous propagation of the key event among other
 components. To not allow any other component to receive the event, return `false`.
 
+Flame also provides a default implementation called `KeyboardListenerComponent` which can be used to handle
+keyboard events. Like any other component, it can be added as a child to a `FlameGame` or another
+`Component`:
```

Comment:
```suggestion
Flame also provides a default implementation called `KeyboardListenerComponent` which can be used to
handle keyboard events. Like any other component, it can be added as a child to a `FlameGame` or
another `Component`:
```